buf-ts.gen.yaml

```diff
@@ -33,10 +28,12 @@ plugins:
       - protoc-gen-ts
     out: gen/proto/ts
     opt:
-      # the next time we tweak the ts codegen we should put the options in
-      # alphabetical order
-      - eslint_disable
       - add_pb_suffix
+      - eslint_disable
+      # By default, only the proto files passed as input to protoc are generated, not the files they
+      # import (with the exception of well-known types which are always generated when imported).
+      # generate_dependencies generates code for dependencies too.
```

Comment:
I believe that the wording here is wrong; we are still passing the whole directory of compiled protos to the plugin like before, this option lets the plugin figure out what to generate based on the list of files we are asking it to generate (in the `paths` list) instead of having to trim down the list ourselves.